RELEASE.md

```diff
@@ -40,7 +40,18 @@ Creating a new release of network proxy involves releasing a new version of the
 
     Once the two tags are created, the konnectivity-client can be imported as a library in kubernetes/kubernetes and other go programs.
 
-2. To publish the proxy server and proxy agent images, they must be promoted from the k8s staging repo. An example PR can be seen here: [https://github.com/kubernetes/k8s.io/pull/1602](https://github.com/kubernetes/k8s.io/pull/1602)
+2. If increasing the minor version (the `y` in `x.y.z`), a new release branch must be created. The name of this branch should be `release-x.y` where `x` is `1` and `y` is the minor release version. For example, if increasing the apiserver-network-proxy from verision `0.98.4` to `0.99.0` a new branch should be created named `release-1.99`.
```

Comment:
My first thought is that release-0.99 would be a better branch name for tags 0.99.x. (But, on the other hand, I see that client-go does it the way you have it here.)

Reply:
yeah, i could go either way on this, i just chose the follow the client-go example for the first pass. if we want use `release-0.99` as our branching name pattern, i'm fine with that.

Reply:
I personally prefer release-0.99, to be more consistent with the tags. But I'm flexible if anybody feels strongly otherwise. The rest LGTM, thanks!

/lgtm
/approve

Let's hold just to allow others time to look.
/hold